web_external/ImagesGallery/imagesColors.styl

```diff
@@ -0,0 +1,9 @@
+$overviewColor = #a8ddb5
+$filteredSetColor = #4eb3d3
+$pageColor = #08589e
+
+$rolloverImageFilter = url(#recolorImageTo00ABFF)
+$selectedImageFilter = url(#recolorImageTo444499)
+$disabledImageFilter = url(#recolorImageToCCCCCC)
```

Comment:
Are all of these still necessary? I believe they were for recoloring SVGs on the fly. IT would be great to remove `isic.shims.recolorImageFilters` altogether if possible.